### DISCUSSION TITLE
Add bookmark must be available on card footer

### DISCUSSION THREAD

**Discussion by mohiyuddinshaikh** [dailydotdev/daily#670] [Feature Requests]
Hi, Love what daily dev is doing. When I joined a month back, bookmark article option was availabe right at the footer of the card. That was very convenient since there are multiple articles that I am interested in but I have got work to do so I saved all of them with just 1 click per card. But now i gotta open the menu and then save bookmark which is kind of an hassle. I am aware I can just pause new articles until i return but bringing back that quick bookmark feature woud be even better.

**Comment by idoshamun** [comment]
Hi @mohiyuddinshaikh,

Thanks for sharing your thoughts. This is a dup of https://github.com/dailydotdev/daily/discussions/638, so let's keep the discussion there

**Comment by nimrodkra** [comment]
@mohiyuddinshaikh we decided to bring back the bookmark button to the card - enjoy! 

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/3e2c4f2d-461e-4ce7-b04b-215d9276de80)
